src/modules/dashboard/components/ActionsPage/FinalizeMotionWidget/VoteResults/VoteResults.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { bigNumberify } from 'ethers/utils';
+
+import { MiniSpinnerLoader } from '~core/Preloaders';
+
+import {
+  Colony,
+  useLoggedInUser,
+  useMotionVoteResultsQuery,
+} from '~data/index';
+
+import VoteResultsItem from './VoteResultsItem';
+
+import styles from './VoteResults.css';
+
+interface Props {
+  colony: Colony;
+  motionId: number;
+}
+
+const MSG = defineMessages({
+  voteYAY: {
+    id: 'dashboard.ActionsPage.FinalizeVoteWidget.VoteResults.voteYAY',
+    defaultMessage: `Yes`,
+  },
+  voteNAY: {
+    id: 'dashboard.ActionsPage.FinalizeVoteWidget.VoteResults.voteNAY',
+    defaultMessage: `No!`,
+  },
+  loadingText: {
+    id: 'dashboard.ActionsPage.FinalizeVoteWidget.VoteResults.loadingText',
```

Comment:
The ids of those MSGs should be `dashboard.ActionsPage.FinalizeMotionWidget.VoteResults ...`, right?

Reply:
Good catch!